File: vb.net/create_email_campaign.vb
Dim client As New RestClient("<REPLACE_WITH_PROD_URL>/v1/campaigns/emails")

client.Timeout = -1

Dim request As New RestRequest(Method.POST)

request.AddHeader("Authorization", "Bearer <REPLACE_WITH_YOUR_API_KEY>")
request.AddHeader("Content-Type", "application/json")
Dim body As String = "{" & vbCrLf &
"    ""label"": ""My Campaign publipostage""," & vbCrLf &
"    ""from": "sender@mail.com""," & vbCrLf &
"    ""recipients"": [" & vbCrLf &
"        ""test@mail.com""," & vbCrLf &
"        ""test@mail.com""," & vbCrLf &
"        ""test@mail.com""," & vbCrLf &
"    ],"  & vbCrLf &
"     ""subject"": ""subject""," & vbCrLf &
"    ""content"": ""Hello {{name}}, votre numéro est {{numero}}""" & vbCrLf &
"}"
request.AddParameter("application/json", body, ParameterType.RequestBody)

Dim response As IRestResponse = client.Execute(request)
Console.WriteLine(response.Content)

File: php/guzzle/create_email_campaign.php
<?php
// GUZZLE
// Make sure to install Guzzle using Composer: composer require guzzlehttp/guzzle

use GuzzleHttp\Client;
use GuzzleHttp\Psr7\Request;

// Replace the following values with your own
$prodUrl = '<REPLACE_WITH_PROD_URL>';
$token = '<REPLACE_WITH_YOUR_API_KEY>';

$client = new Client();
$headers = [
    'Authorization' => 'Bearer ' . $token,
    'Content-Type' => 'application/json'
];
$body = '{
  "label": "My Campaign publipostage",
    "from": "sender@mail.com",
    "recipients": ["test@mail.com", "test@mail.com", "test@mail.com"],
    "content": "Hello {{name}}, votre numéro est {{numero}}",
    "subject": "subject",
}';
$request = new Request('POST', $prodUrl . '/v1/campaigns/emails', $headers, $body);
$res = $client->sendAsync($request)->wait();
echo $res->getBody();

